<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetDialog</class>
 <widget class="QDialog" name="SetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>580</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>580</height>
    </rect>
   </property>
   <widget class="QWidget" name="containerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>297</x>
      <y>161</y>
      <width>365</width>
      <height>257</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>60</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(190,190,190);</string>
     </property>
     <property name="text">
      <string>设置</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QToolButton" name="generalBtn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>49</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>通用</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>365</width>
       <height>177</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QToolButton" name="saveBtn">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>147</y>
         <width>73</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton{color:white;border:1px solid  rgb(46,78,126);border-radius:6px;
background-color: rgb(46,78,126);}
QToolButton:hover{background-color:rgb(66,98,146);}
QToolButton:disabled{background-color:#cecece;}</string>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>6</y>
         <width>131</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(190, 190, 190);</string>
       </property>
       <property name="text">
        <string>钱包自动锁定时间</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="languageComboBox">
       <property name="geometry">
        <rect>
         <x>146</x>
         <y>42</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <item>
        <property name="text">
         <string>中文</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="nolockCheckBox">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>6</y>
         <width>60</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>不锁定</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>41</x>
         <y>42</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(190, 190, 190);</string>
       </property>
       <property name="text">
        <string>界面语言</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>6</y>
         <width>31</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(190, 190, 190);</string>
       </property>
       <property name="text">
        <string>min</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="minimizeCheckBox">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>78</y>
         <width>171</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>最小化托盘</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="closeCheckBox">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>104</y>
         <width>211</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>点击关闭按钮最小化</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lockTimeLineEdit">
       <property name="geometry">
        <rect>
         <x>146</x>
         <y>6</y>
         <width>81</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(37,39,43);
color: rgb(190, 190, 190);
border:none;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QLineEdit" name="newPwdLineEdit">
       <property name="geometry">
        <rect>
         <x>132</x>
         <y>42</y>
         <width>143</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLineEdit" name="confirmPwdLineEdit">
       <property name="geometry">
        <rect>
         <x>132</x>
         <y>78</y>
         <width>143</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QToolButton" name="confirmBtn">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>147</y>
         <width>73</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton{color:white;border:1px solid  rgb(46,78,126);border-radius:6px;
background-color: rgb(46,78,126);}
QToolButton:hover{background-color:rgb(66,98,146);}
QToolButton:disabled{background-color:#cecece;}</string>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>78</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(190, 190, 190);</string>
       </property>
       <property name="text">
        <string>再次输入：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="tipLabel3">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>6</y>
         <width>85</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>6</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(190, 190, 190);</string>
       </property>
       <property name="text">
        <string>旧密码：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="tipLabel2">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>78</y>
         <width>85</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>42</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(190, 190, 190);</string>
       </property>
       <property name="text">
        <string>新密码：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="tipLabel">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>42</y>
         <width>85</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="oldPwdLineEdit">
       <property name="geometry">
        <rect>
         <x>132</x>
         <y>6</y>
         <width>143</width>
         <height>24</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QToolButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>336</x>
       <y>10</y>
       <width>12</width>
       <height>11</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="safeBtn">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>49</y>
       <width>60</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>oldPwdLineEdit</tabstop>
  <tabstop>newPwdLineEdit</tabstop>
  <tabstop>confirmPwdLineEdit</tabstop>
  <tabstop>generalBtn</tabstop>
  <tabstop>safeBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
